sql-statements/sql-statement-show-collation.md

```diff
@@ -9,256 +9,61 @@ aliases: ['/docs-cn/dev/reference/sql/statements/show-collation/']
 
 `SHOW COLLATION` 语句用于提供一个静态的排序规则列表，确保与 MySQL 客户端库的兼容性。
 
-> **注意:** TiDB 目前仅支持二进制排序规则。
+> **注意:** `SHOW COLLATION` 所展示的排序规则列表与是否 TiDB 集群是否开启新排序规则框架有关，详情请见 [TiDB 字符集和排序规则](/character-set-and-collation.md))
 
 ## 语法图
 
-**ShowStmt:**
+**ShowCollationStmt:**
 
-![ShowStmt](/media/sqlgram/ShowStmt.png)
-
-**ShowTargetFilterable:**
-
-![ShowTargetFilterable](/media/sqlgram/ShowTargetFilterable.png)
+![ShowCollationStmt](/media/sqlgram/ShowCollationStmt.png)
 
 ## 示例
 
 {{< copyable "sql" >}}
 
 ```sql
+" 若未开启新排序规则框架，仅展示二进制排序规则
```

Comment:
```suggestion
```

Reply:
It is not recommended to add comments into code blocks. Please move the comments to L22~